docs/guides/Quick Start/04-document-management.md

```diff
@@ -0,0 +1,73 @@
+# Overview
+
+One can use documents to send invoive, delivery note, packing slips, etc. Already defined document types can be fetched from 
```

Comment:
Maybe we can mention here, that some documents are sent in an "automated" way - e.g. when an order transitions into a specific state. I assume people will hit this page, expecting to read how to send documents in an automated way - yet the guide only deals with document types and downloads. I think a small hint / connection to that would be helpful.

Reply:
okay 👍